github_activity/github_activity.py

```diff
@@ -170,7 +170,13 @@ def generate_activity_md(target, since=None, until=None, kind=None, auth=None,
     for _, iitems in data.iterrows():
         item_commentors = []
         for icomment in iitems['comments']['edges']:
-            comment_author = icomment['node']['author']['login']
+            comment_author = icomment['node']['author']
+            if comment_author == None:
```

Comment:
could we collapse this into just:

```
if comment_author:
    do_stuff
```
?

Reply:
I didn't grasp fully what you suggested so I gave it an open rethink myself and fixed it to what I think I'd like and I'll leave it to you to do additional refactoring. I have no worries about such!